test/fake/eckey.go

```diff
@@ -26,3 +29,15 @@ func ReadECKey(filename string) (*ecdsa.PrivateKey, error) {
 
 	return x509.ParseECPrivateKey(privateKeyBlock.Bytes)
 }
+
+func ReadFileFromProjectDirectory(filename string) ([]byte, error) {
+	_, b, _, ok := runtime.Caller(0)
+	if !ok {
+		return nil, fmt.Errorf("reading file from project directory: %w", errFailedToGetCaller)
+	}
+
+	root := filepath.Join(filepath.Dir(b), "..", "..") // from ./test/fake to project directory
+	path := filepath.Clean(filepath.Join(root, filename))
+
+	return os.ReadFile(path)
+}
```

Comment:
Could we look into if we could use [Go `embed` directive](https://pkg.go.dev/embed) instead?